<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenCVOutlinesWindowBase</class>
 <widget class="QMainWindow" name="OpenCVOutlinesWindowBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="imageTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="sourceTab">
       <attribute name="title">
        <string>Source</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout"/>
      </widget>
      <widget class="QWidget" name="contourTab">
       <attribute name="title">
        <string>Contours</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Process</string>
    </property>
    <addaction name="actionOutlines"/>
    <addaction name="actionCorners"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionOutlines">
   <property name="text">
    <string>Locate Outlines</string>
   </property>
  </action>
  <action name="actionCorners">
   <property name="text">
    <string>Locate Corners</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
